x-pack/legacy/plugins/integrations_manager/common/constants.ts

```diff
@@ -7,3 +7,18 @@ import manifest from '../kibana.json';
 
 export const PLUGIN_ID = manifest.id;
 export const SAVED_OBJECT_TYPE = 'integrations-manager';
+
+export enum AssetTypes {
```

Comment:
The [public Saved Object types](https://www.elastic.co/guide/en/kibana/current/saved-objects-api-get.html#_path_parameters)